README.rst

```diff
@@ -1,130 +1,165 @@
-to create rules which a project's code should conform to. *coala* takes care
-of showing these issues to users in a friendly manner, is versatile and can be
-used in any environment. Patches to automatically fix code will be managed too.
-*coala* has a set of official bears (plugins) to provide an out-of-the-box
-analysis functionality for many popular languages in addition to some
-generically applicable algorithms.
+============
+Installation
+============
 
-To see what coala can do for you and your language, take a look at
-`our capabilities listing <https://github.com/coala-analyzer/bear-docs/blob/master/README.rst>`__.
+To install the **latest stable version** run:
 
-*coala* is written with a lower case "c".
+.. code-block:: bash
 
-Read more at our `documentation <http://coala.rtfd.org/>`__.
+    $ pip3 install coala-bears
 
-Why use coala?
---------------
+To install the latest development version run:
 
-- *coala* provides **built-in checking routines** (named bears in *coala*).
-- **Serves your requirement**: You can easily write your own checks (using
-  bears).
-- *coala* provides **user-friendly interfaces** like json, formatted and
-  interactive output in the CLI and *plugins for various editors* are
-  available as well.
-- **Optimal performance**: *coala* manages parallelizing the checking-routines
-  without you having to worry.
-- **Unified interface**: One tool for all programming languages.
+.. code-block:: bash
 
-If you want to learn more about *coala*, its functionality and its usage,
-please take a look at our
-`tutorial <http://coala.rtfd.org/en/latest/Users/Tutorials/Tutorial.html>`__.
+    $ pip3 install coala-bears --pre
 
-Installation
-------------
+|PyPI| |Windows| |Linux|
 
-To install *coala* with the official set of analysis routines you can simply run
-``pip3 install coala-bears``.
+-----
 
-*coala* only, without the official bears, can be installed with
-``pip3 install coala``. If you need more information about the installation and
-dependencies, take a look at our `installation documentation
-<http://coala.rtfd.org/en/latest/Users/Install.html>`__.
+=====
+Usage
+=====
 
-The latest code from master is automatically deployed to PyPI as a
-development version. Get it with ``pip3 install coala --pre``.
+To get the list of arguments and their meaning, run:
 
-|PyPI|
+.. code-block:: bash
 
-Authors
--------
+    $ coala --help
 
-*coala* is maintained by a growing community. Please take a look at the
-meta information in `setup.py <setup.py>`__ for current maintainers.
+To run coala over your project, create a ``.coafile``, the project-specific
+configuration file that will store all your settings for coala. A sample
+``.coafile`` will look something like this:
 
-Project Status
---------------
+.. code-block:: bash
 
-|Linux Build Status| |Windows Build status|
+    [Default]
+    files = src/**/*.py
+    bears = SpaceConsistencyBear
+    use_spaces = True
```

Comment:
this section could use some love. I mean, here's how we teach newcomers how to use coala, eh?

first of all, dont name it Default. secondly, explain it after, briefly.

[Spacing]
files = src/**/*.py
bears = SpaceConsistencyBear
use_spaces = True

and then explain:
``Where Spacing is the name of the section, files is the files you would like to run coala on, bears are the bears that are to be used, and use_spaces specifies the bear to use spaces instead of tabs.
After writing this section, simply run "$ coala spacing" and you are done.``